main.cpp

```diff
@@ -152,6 +169,11 @@ void TopRopeRoute::RouteGrade::printGradeInfo()
     std::cout << "Grade is: " << gradeNumber << gradeLetter << std::endl;
 }
 
+void TopRopeRoute::RouteGrade::printGradeInfo_()    // new function
+{
+    std::cout << "Grade is: " << this->gradeNumber << this->gradeLetter << std::endl;
+}
```

Comment:
- [ ] see previous instructions/comments about adding `this->`
- [ ] remove this function
- [ ] revise previous version